src/content/docs/en/guides/deploy/microsoft-azure.mdx

```diff
@@ -21,6 +21,6 @@ To follow this guide you will need:
 
 2. Open the Static Web Apps extension, sign in to Azure, and click the **+** button to create a new Static Web App. You will be prompted to designate which subscription key to use.
 
-3. Follow the wizard started by the extension to give your app a name, choose a framework preset, and designate the app root (usually `/`) and built file location `/dist`. The wizard will run and will create a [GitHub Action](https://github.com/features/actions) in your repo in a `.github` folder.
+3. Follow the wizard started by the extension to give your app a name, choose a framework preset, and designate the app root (usually `/`) and built file location `/dist`. The wizard will run and will create a [GitHub Action](https://github.com/features/actions) the `.github` folder of your repo.
```

Comment:
```suggestion
3. Follow the wizard started by the extension to give your app a name, choose a framework preset, and designate the app root (usually `/`) and built file location `/dist`. The wizard will run and will create a [GitHub Action](https://github.com/features/actions) in the `.github` folder of your repo. (This folder will be automatically created if it does not already exist.)
```
Just a missing word here! And, using "the" instead of "a" now assumes that there already *is* a `.github` folder in your project, which there may not be, so I added a note. What do you think?